gems/ruby-saml/CVE-2016-5697.yml

```diff
@@ -0,0 +1,10 @@
+---
+gem: ruby-saml
+framework: rails
```

Comment:
`ruby-saml` isn't part of `rails` directly, as far as I am aware. Please remove.